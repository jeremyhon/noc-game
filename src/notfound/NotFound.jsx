import React from "react"

export default class NotFound extends React.Component {
  static propTypes = {

  }

  render() {
    return (
      <div>
        {"Oops! This page doesn't exist!"}
      </div>
    )
  }
}
