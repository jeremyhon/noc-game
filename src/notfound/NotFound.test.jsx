import React from 'react'
import { shallow } from 'enzyme'

import NotFound from './NotFound'

it("matches the snapshot", () => {
  expect(shallow(<NotFound />)).toMatchSnapshot()
})
