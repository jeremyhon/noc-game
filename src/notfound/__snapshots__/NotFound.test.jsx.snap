// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div>
  <br />
  "Oops! This page doesn't exist!"
  <br />
  Click 
  <a
    href="/noc-game"
  >
    here
  </a>
   to go back.
</div>
`;
